mcs_benchmark_data/pipelines/commonsense_qa/commonsense_qa_benchmark_transformer.py

```diff
@@ -81,19 +77,19 @@ def __transform_benchmark_sample(
         question_type = BenchmarkQuestionType.MULTIPLE_CHOICE
         question_category = None
 
-        ans_mapping = {ans: i for i, ans in enumerate("ABCDE")}
-
         for line in all_samples:
 
             sample = json.loads(line)
 
-            correct_choice = None
+            correct_choice = URIRef("")
```

Comment:
None was correct.

Reply:
When I used None, it returned the error "Object None must be an rdflib term." Is there a way to work around this?